include/seqan3/range/view/take_until.hpp

```diff
@@ -572,6 +572,11 @@ namespace seqan3::view
  *
  * \details
  *
+ * **Header**
+ * ```cpp
+ *      #include <seqan3/range/view/take_until.hpp>
+ * ```
+ *
```

Comment:
There are multiple view adaptor definitions in this file, I think you forgot the others? Please also double-check the other files.

Reply:
No, I checked all Headers are in the definition, for some files there seems to be a general documentation part because it is the same for all functions.

Reply:
What about view::take_exactly_or_throw? What about view::take_until_and_consume? ....

Reply:
Checked all view functions, all had a header.

Reply:
Ah, then that's resolved because of the \copydetails